gideon.py

```diff
@@ -118,11 +122,13 @@ def init_kv_parser():
     kv_parser.add_argument("--expire",  help="percentage of expirations 0-100", default=0, type=int)
     kv_parser.add_argument("--ttl",      default=15, help="document expires time to use when expirations set")
     kv_parser.add_argument("--delete",  help="percentage of deletes 0-100", default=0, type=int)
-    kv_parser.add_argument("--template",help="predefined template to use", default="default")
+    kv_parser.add_argument("--template", help="predefined template to use", default="default")
```

Comment:
Nit: This is a whitespace-only change unrelated to this patch, which will break tools like `git annotate` etc to determine what/why a line was last changed. Suggest you revert it.

Reply:
sure reverting it.